### PR TITLE
Create device smart_ac_outlet_single_key.yaml

### PR DESCRIPTION
**Pull request by IamGimli**
New device definition file for generic smart outlet

### CODE REVIEW

**Comment by make-all** [comment]
"generic" should only be used for common dps patterns that match a lot of devices. This is an unusal dps pattern, and probably unique to this specific device. So manufacturer and model should be specified as part of the filename (and in the products section)

**Comment by IamGimli** [comment]
Alright, I believe the manufacturer is CE and the model number is LQ-2-W3, commonly marketed as "Wi-Fi Smart Outlet". I'll rename the file.

The product name that's in the file under the "products" header is what is reported by the device to Tuya, should I also update that or leave it as-is?

**Comment by make-all** [comment]
CE - you are not looking at the CE certification mark?

**Comment by IamGimli** [comment]
That's correct.

File: custom_components/tuya_local/devices/smart_ac_outlet_single_key.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+     dps:
+       - id: 102
+         name: value
+         type: integer
+         unit: min
+         range:
+           min: 0
+           max: 86400
+         mapping:
+           - scale: 0
```
[the remark this answers is not in the document]
Comment (IamGimli) [reply custom_components/tuya_local/devices/smart_ac_outlet_single_key.yaml:24]: Interesting, I was just using what seems to be reported by the device itself:

>        "type": "Integer",
          "values": "{\"unit\":\"\",\"min\":0,\"max\":86400,\"scale\":0,\"step\":1}"

[the remark this answers is not in the document]
Comment (IamGimli) [reply custom_components/tuya_local/devices/smart_ac_outlet_single_key.yaml]: Made changes to address what we discussed so far, please let me know if there's anything else that seem incorrect.

**Pull request merged**